pandas/core/groupby/generic.py

```diff
@@ -1278,7 +1277,12 @@ def _wrap_applied_output(self, keys, values, not_indexed_same=False):
 
         return self._reindex_output(result)
 
-    def _transform_general(self, func, *args, **kwargs):
+    def _transform_general(
```

Comment:
hmm, can you rebase IIRC we removed the engine_kwargs so this signature can be reverted

Reply:
yes - made a mistake resolving merge conflict. Reverted